common/src/main/java/whocraft/tardis_refined/common/block/device/TerraformerBlock.java

```diff
@@ -126,16 +126,19 @@ public InteractionResult use(BlockState blockState, Level level, BlockPos blockP
         if (level instanceof ServerLevel serverLevel) {
             if (interactionHand == InteractionHand.MAIN_HAND) {
                 if (player.getMainHandItem().getItem() instanceof ScrewdriverItem screwdriverItem) {
-                    screwdriverItem.playScrewdriverSound(serverLevel, blockPos, SoundRegistry.SCREWDRIVER_SHORT.get());
+                    screwdriverItem.playScrewdriverSound(serverLevel, blockPos, TRSoundRegistry.SCREWDRIVER_SHORT.get());
 
                     if (checkIfStructure(serverLevel, blockPos)) {
                         TardisLevelOperator.get(serverLevel).ifPresent(cap -> {
                             TardisInteriorManager interiorManager = cap.getInteriorManager();
                             if (interiorManager.isWaitingToGenerate()) {
                                 level.destroyBlock(blockPos, true);
                             } else {
-                                if (interiorManager.isCave()) {
-                                    interiorManager.prepareDesktop(TardisDesktops.FACTORY_THEME);
+
+                                System.out.println("We've determined it's good G");
+                                System.out.println(cap.getTardisState());
```

Comment:
Remove debugging system print statements or wrap in a check for development environment